src/tabs/tab-headers.component.ts

```diff
@@ -290,8 +296,17 @@ export class TabHeaders implements AfterContentInit, OnChanges, OnInit {
 		}
 	}
 
-	ngOnInit() {
-		this.eventService.on(window as any, "resize", () => this.handleScroll());
+	ngOnInit(): void {
+		// Update scroll on resize
+		this.resizeObserver = new ResizeObserver(() => {
+			// Need to explicitly trigger change detection since this runs outside Angular zone
+			this.changeDetectorRef.detectChanges();
+		});
+		this.resizeObserver.observe(this.tabsScrollable.nativeElement);
+	}
```

Comment:
Is `this.eventService.on(window as any, "resize", () => this.handleScroll());` not working? 🤔 

What browser are you using?

Reply:
@Akshat55 It doesn't matter which browser; the problem is that we need to handle view resize, not window resize. For example, if we use a tabs component on a side panel which can be dynamically resized, the tabs will be resized accordingly. But we don't get resize event on window.

Reply:
You can simulate the problem by manually setting the width of the parent container in DevTools. Since we set the width in DevTools, the Angular component tree will not receive any user event and therefore the Angular change detection won't be triggered. Therefore the tab scroll won't be updated properly.

<img width="1535" alt="image" src="https://user-images.githubusercontent.com/34791554/227531026-647b6da5-9981-4b52-927e-0197c169b9a1.png">

In most cases, there's some user event happens on the component tree while the view is resized and the Angular change detection will be triggered, so this is not a problem.

However, in some cases, the view might be resized when there's not a user event on the component tree. For example, during open/expand/collapse animation. This requires us to handle view resize explicitly and trigger change detection.